scripts/RNA_analysis/seting_dependensies.sh

```diff
@@ -22,7 +22,9 @@ gtf_file=$7
 WP=$8
```

Comment:
How much trouble would it be to rename this file? The spelling mistakes upset me every time I see it.. :wink:  `seting_dependensies.sh` -> `setting_dependencies.sh`